LoopKitUI/Views/QuantityPicker.swift

```diff
@@ -101,20 +101,33 @@ public struct QuantityPicker: View {
     }
 
     public var body: some View {
-        Picker("Quantity", selection: selectedValue) {
-            ForEach(selectableValues, id: \.self) { value in
-                Text(self.formatter.string(from: value) ?? "\(value)")
-                    .foregroundColor(self.colorForValue(value))
-                    .anchorPreference(key: PickerValueBoundsKey.self, value: .bounds, transform: { [$0] })
-                    .accessibility(identifier: self.formatter.string(from: value) ?? "\(value)")
+        picker
+            .labelsHidden()
+            .pickerStyle(WheelPickerStyle())
+            .overlayPreferenceValue(PickerValueBoundsKey.self, unitLabel(positionedFrom:))
+            .accessibility(identifier: "quantity_picker")
+    }
+
+    @ViewBuilder
+    private var picker: some View {
+        if useSizeablePicker {
+            SizeablePicker(selection: selectedValue,
+                           data: selectableValues,
+                           formatter: { self.formatter.string(from: $0) ?? "\($0)" },
+                           colorer: colorForValue)
+                .anchorPreference(key: PickerValueBoundsKey.self, value: .bounds, transform: { [$0] })
+        } else {
+            Picker("Quantity", selection: selectedValue) {
```

Comment:
[nit] Localized?

Reply:
This code is going away. But, also, this string is actually never used...